src/com/google/common/css/JobDescriptionBuilder.java

```diff
@@ -370,10 +382,20 @@ public JobDescriptionBuilder setAllowKeyframes(boolean allow) {
     return this;
   }
 
+  public JobDescriptionBuilder setAllowSupports(boolean allow) {
+    checkJobIsNotAlreadyCreated();
+    this.allowSupports = allow;
+    return this;
+  }
+
   public JobDescriptionBuilder allowKeyframes() {
     return setAllowKeyframes(true);
   }
 
+  public JobDescriptionBuilder allowSupports() {
+    return setAllowSupports(true);
```

Comment:
enable `allowSupports` to be called on `JobDescriptionBuilder`, just like `allowKeyframes`